ktx-mapbox-maps/gradle.properties

```diff
@@ -0,0 +1,5 @@
+VERSION_NAME=0.1.0-SNAPSHOT
+POM_ARTIFACT_ID=mapbox-android-sdk-ktx
+POM_NAME=Mapbox Maps KTX
```

Comment:
How about `Mapbox Maps SDK for Android KTX` to keep it consistent with the maps SDK?